### ISSUE TITLE
Perf Dashboard: Tracing doesn't seem to add traces

### ISSUE THREAD

**Issue by quisquous** [catapult-project/catapult#2485]
Chrome version: 51.0.2704.106 (Linux)
URL: https://chromeperf.appspot.com/report?sid=74f6fb42f691bb1e6d024a7e742df324a23596684c089837448742004d9d3d3f&rev=401811

I clicked on the graph on the exclamation, hit the Trace button, and expected to have that trace be added to the exclamation point's dialog.  Other times I've clicked on graphs on the perf dashboard, I've been able to download traces.  The try job seemed to completely correctly as well: https://codereview.chromium.org/2143973006

Maybe I'm just holding it wrong?

**Comment by anniesullie** [comment]
tl;dr: your traces are at:
https://console.developers.google.com/m/cloudstorage/b/chrome-telemetry-output/o/profiler-file-id_4-2016-07-14_14-34-5672152.zip
https://console.developers.google.com/m/cloudstorage/b/chrome-telemetry-output/o/profiler-file-id_4-2016-07-14_14-38-3072645.zip

Long version:
I'm really sorry, the UI for the tracing features on the dashboard isn't great. It has two different features:

* For newer benchmarks which are trace-based, it gives a link to the trace the metrics were computed from during the original run on the perf waterfall. blink_perf.svg is not one of those benchmarks.
* For all benchmarks, you can press the "Trace" button and it'll run a perf tryjob and upload the traces, but it's hard to get to the traces. You need to click through the code review to the try job: https://build.chromium.org/p/tryserver.chromium.perf/builders/mac_10_11_perf_bisect/builds/740 and then look at the output of the "Performance Test" steps:

```
@@@STEP_LOG_LINE@Captured Output@(INFO) 2016-07-14 14:38:23,052 cloud_storage.Insert:292  Uploading /b/build/slave/mac_10_11_perf_bisect/build/src/tools/perf/DropsOnABlade_html.zip to gs://chrome-telemetry-output/profiler-file-id_7-2016-07-14_14-38-2321643.zip@@@
@@@STEP_LOG_LINE@Captured Output@View generated profiler files online at https://console.developers.google.com/m/cloudstorage/b/chrome-telemetry-output/o/profiler-file-id_7-2016-07-14_14-38-2321643.zip for page DropsOnABlade.html@@@
@@@STEP_LOG_LINE@Captured Output@(INFO) 2016-07-14 14:38:24,582 cloud_storage.Insert:292  Uploading /b/build/slave/mac_10_11_perf_bisect/build/src/tools/perf/Worldcup_html.zip to gs://chrome-telemetry-output/profiler-file-id_15-2016-07-14_14-38-2495708.zip@@@
@@@STEP_LOG_LINE@Captured Output@View generated profiler files online at https://console.developers.google.com/m/cloudstorage/b/chrome-telemetry-output/o/profiler-file-id_15-2016-07-14_14-38-2495708.zip for page Worldcup.html@@@
@@@STEP_LOG_LINE@Captured Output@(INFO) 2016-07-14 14:38:25,586 cloud_storage.Insert:292  Uploading /b/build/slave/mac_10_11_perf_bisect/build/src/tools/perf/SvgNestedUse_html.zip to gs://chrome-telemetry-output/profiler-file-id_13-2016-07-14_14-38-2590650.zip@@@
@@@STEP_LOG_LINE@Captured Output@View generated profiler files online at https://console.developers.google.com/m/cloudstorage/b/chrome-telemetry-output/o/profiler-file-id_13-2016-07-14_14-38-2590650.zip for page SvgNestedUse.html@@@
@@@STEP_LOG_LINE@Captured Output@(INFO) 2016-07-14 14:38:26,556 cloud_storage.Insert:292  Uploading /b/build/slave/mac_10_11_perf_bisect/build/src/tools/perf/Debian_html.zip to gs://chrome-telemetry-output/profiler-file-id_6-2016-07-14_14-38-2617948.zip@@@
@@@STEP_LOG_LINE@Captured Output@View generated profiler files online at https://console.developers.google.com/m/cloudstorage/b/chrome-telemetry-output/o/profiler-file-id_6-2016-07-14_14-38-2617948.zip for page Debian.html@@@
@@@STEP_LOG_LINE@Captured Output@(INFO) 2016-07-14 14:38:27,419 cloud_storage.Insert:292  Uploading /b/build/slave/mac_10_11_perf_bisect/build/src/tools/perf/SvgHitTesting_html.zip to gs://chrome-telemetry-output/profiler-file-id_12-2016-07-14_14-38-2782947.zip@@@
@@@STEP_LOG_LINE@Captured Output@View generated profiler files online at https://console.developers.google.com/m/cloudstorage/b/chrome-telemetry-output/o/profiler-file-id_12-2016-07-14_14-38-2782947.zip for page SvgHitTesting.html@@@
@@@STEP_LOG_LINE@Captured Output@(INFO) 2016-07-14 14:38:28,822 cloud_storage.Insert:292  Uploading /b/build/slave/mac_10_11_perf_bisect/build/src/tools/perf/Cactus_html.zip to gs://chrome-telemetry-output/profiler-file-id_2-2016-07-14_14-38-2866628.zip@@@
@@@STEP_LOG_LINE@Captured Output@View generated profiler files online at https://console.developers.google.com/m/cloudstorage/b/chrome-telemetry-output/o/profiler-file-id_2-2016-07-14_14-38-2866628.zip for page Cactus.html@@@
@@@STEP_LOG_LINE@Captured Output@(INFO) 2016-07-14 14:38:29,757 cloud_storage.Insert:292  Uploading /b/build/slave/mac_10_11_perf_bisect/build/src/tools/perf/SvgCubics_html.zip to gs://chrome-telemetry-output/profiler-file-id_11-2016-07-14_14-38-2989108.zip@@@
@@@STEP_LOG_LINE@Captured Output@View generated profiler files online at https://console.developers.google.com/m/cloudstorage/b/chrome-telemetry-output/o/profiler-file-id_11-2016-07-14_14-38-2989108.zip for page SvgCubics.html@@@
@@@STEP_LOG_LINE@Captured Output@(INFO) 2016-07-14 14:38:30,551 cloud_storage.Insert:292  Uploading /b/build/slave/mac_10_11_perf_bisect/build/src/tools/perf/Cowboy_transform_html.zip to gs://chrome-telemetry-output/profiler-file-id_4-2016-07-14_14-38-3072645.zip@@@
@@@STEP_LOG_LINE@Captured Output@View generated profiler files online at https://console.developers.google.com/m/cloudstorage/b/chrome-telemetry-output/o/profiler-file-id_4-2016-07-14_14-38-3072645.zip for page Cowboy_transform.html@@@
@@@STEP_LOG_LINE@Captured Output@(INFO) 2016-07-14 14:38:31,443 cloud_storage.Insert:292  Uploading /b/build/slave/mac_10_11_perf_bisect/build/src/tools/perf/FlowerFromMyGarden_html.zip to gs://chrome-telemetry-output/profiler-file-id_8-2016-07-14_14-38-3168410.zip@@@
@@@STEP_LOG_LINE@Captured Output@View generated profiler files online at https://console.developers.google.com/m/cloudstorage/b/chrome-telemetry-output/o/profiler-file-id_8-2016-07-14_14-38-3168410.zip for page FlowerFromMyGarden.html@@@
@@@STEP_LOG_LINE@Captured Output@(INFO) 2016-07-14 14:38:32,271 cloud_storage.Insert:292  Uploading /b/build/slave/mac_10_11_perf_bisect/build/src/tools/perf/CrawFishGanson_html.zip to gs://chrome-telemetry-output/profiler-file-id_5-2016-07-14_14-38-3231300.zip@@@
@@@STEP_LOG_LINE@Captured Output@View generated profiler files online at https://console.developers.google.com/m/cloudstorage/b/chrome-telemetry-output/o/profiler-file-id_5-2016-07-14_14-38-3231300.zip for page CrawFishGanson.html@@@
@@@STEP_LOG_LINE@Captured Output@(INFO) 2016-07-14 14:38:33,090 cloud_storage.Insert:292  Uploading /b/build/slave/mac_10_11_perf_bisect/build/src/tools/perf/WorldIso_html.zip to gs://chrome-telemetry-output/profiler-file-id_16-2016-07-14_14-38-3339933.zip@@@
@@@STEP_LOG_LINE@Captured Output@View generated profiler files online at https://console.developers.google.com/m/cloudstorage/b/chrome-telemetry-output/o/profiler-file-id_16-2016-07-14_14-38-3339933.zip for page WorldIso.html@@@
@@@STEP_LOG_LINE@Captured Output@(INFO) 2016-07-14 14:38:33,948 cloud_storage.Insert:292  Uploading /b/build/slave/mac_10_11_perf_bisect/build/src/tools/perf/FoodLeifLodahl_html.zip to gs://chrome-telemetry-output/profiler-file-id_9-2016-07-14_14-38-3334756.zip@@@
@@@STEP_LOG_LINE@Captured Output@View generated profiler files online at https://console.developers.google.com/m/cloudstorage/b/chrome-telemetry-output/o/profiler-file-id_9-2016-07-14_14-38-3334756.zip for page FoodLeifLodahl.html@@@
@@@STEP_LOG_LINE@Captured Output@(INFO) 2016-07-14 14:38:34,785 cloud_storage.Insert:292  Uploading /b/build/slave/mac_10_11_perf_bisect/build/src/tools/perf/Bamboo_html.zip to gs://chrome-telemetry-output/profiler-file-id_1-2016-07-14_14-38-3418796.zip@@@
@@@STEP_LOG_LINE@Captured Output@View generated profiler files online at https://console.developers.google.com/m/cloudstorage/b/chrome-telemetry-output/o/profiler-file-id_1-2016-07-14_14-38-3418796.zip for page Bamboo.html@@@
@@@STEP_LOG_LINE@Captured Output@(INFO) 2016-07-14 14:38:35,817 cloud_storage.Insert:292  Uploading /b/build/slave/mac_10_11_perf_bisect/build/src/tools/perf/AzLizardBenjiPark_html.zip to gs://chrome-telemetry-output/profiler-file-id_0-2016-07-14_14-38-3585775.zip@@@
@@@STEP_LOG_LINE@Captured Output@View generated profiler files online at https://console.developers.google.com/m/cloudstorage/b/chrome-telemetry-output/o/profiler-file-id_0-2016-07-14_14-38-3585775.zip for page AzLizardBenjiPark.html@@@
@@@STEP_LOG_LINE@Captured Output@(INFO) 2016-07-14 14:38:36,625 cloud_storage.Insert:292  Uploading /b/build/slave/mac_10_11_perf_bisect/build/src/tools/perf/UnderTheSee_html.zip to gs://chrome-telemetry-output/profiler-file-id_14-2016-07-14_14-38-3681696.zip@@@
@@@STEP_LOG_LINE@Captured Output@View generated profiler files online at https://console.developers.google.com/m/cloudstorage/b/chrome-telemetry-output/o/profiler-file-id_14-2016-07-14_14-38-3681696.zip for page UnderTheSee.html@@@
@@@STEP_LOG_LINE@Captured Output@(INFO) 2016-07-14 14:38:37,402 cloud_storage.Insert:292  Uploading /b/build/slave/mac_10_11_perf_bisect/build/src/tools/perf/Cowboy_html.zip to gs://chrome-telemetry-output/profiler-file-id_3-2016-07-14_14-38-3742187.zip@@@
@@@STEP_LOG_LINE@Captured Output@View generated profiler files online at https://console.developers.google.com/m/cloudstorage/b/chrome-telemetry-output/o/profiler-file-id_3-2016-07-14_14-38-3742187.zip for page Cowboy.html@@@
@@@STEP_LOG_LINE@Captured Output@(INFO) 2016-07-14 14:38:38,278 cloud_storage.Insert:292  Uploading /b/build/slave/mac_10_11_perf_bisect/build/src/tools/perf/HarveyRayner_html.zip to gs://chrome-telemetry-output/profiler-file-id_10-2016-07-14_14-38-3874841.zip@@@
@@@STEP_LOG_LINE@Captured Output@View generated profiler files online at https://console.developers.google.com/m/cloudstorage/b/chrome-telemetry-output/o/profiler-file-id_10-2016-07-14_14-38-3874841.zip for page HarveyRayner.html@@@
```

Frustratingly, we can't make the logs public for release official builds, which we want to use because they perform like what users see. The buildbot urls are finicky when we make them google-only, so you may have to replace `build.chromium.org/p` with `uberchromegw/i` to get logs.

Again I'm sorry this is such a mess. We're working on cleaning it up, but it's slow-going.

**Comment by quisquous** [comment]
Thanks for the help!
